src/acl.c

```diff
@@ -1807,19 +1804,20 @@ void ACLFreeLogEntry(void *leptr) {
  *
  * The argpos argument is used when the reason is ACL_DENIED_KEY or 
  * ACL_DENIED_CHANNEL, since it allows the function to log the key or channel
- * name that caused the problem. If this argument is equal to -1, the function will log the value stored in object.
- * Similarly, if the username argument is provided (not NULL), it will be logged, otherwise the value will be
- * extracted from c->user->name.
+ * name that caused the problem.
+ *
+ * The last 3 arguments are a manual override to be used, instead of any of the automatic ones which
+ * depend on the c and reason arguments.
```

Comment:
```suggestion
 * The last 3 arguments are a manual override to be used, instead of any of the automatic
 * ones which depend on the client and reason arguments (use NULL and -1 for default).
```